lib/clients/websocket.js

```diff
@@ -1,111 +1,87 @@
-var EventEmitter = require('events').EventEmitter;
-var Websocket = require('ws');
-var Utils = require('../utilities.js');
-var util = require('util');
-var _  = {assign: require('lodash.assign')};
-var signRequest = require('../../lib/request_signer').signRequest;
+const EventEmitter = require('events').EventEmitter;
+const Websocket = require('ws');
+const Utils = require('../utilities.js');
+const signRequest = require('../../lib/request_signer').signRequest;
```

Comment:
Destructuring for `EventEmitter` and `signRequest` — should also be enforced by eslint (`prefer-destructuring`).